third_party/2and3/cryptography/x509.pyi

```diff
@@ -296,6 +305,80 @@ class SubjectAlternativeName(ExtensionType):
     def __iter__(self) -> Generator[GeneralName, None, None]: ...
     def get_values_for_type(self, type: Type[GeneralName]) -> List[Any]: ...
 
+class AuthorityKeyIdentifier(ExtensionType):
+    key_identifier: bytes
+    authority_cert_issuer: List[GeneralName]
+    authority_cert_serial_number: Optional[int]
+    def __init__(
+        self,
+        key_identifier: bytes,
+        authority_cert_issuer: List[GeneralName],
+        authority_cert_serial_number: Optional[int],
+    ) -> None: ...
+    @classmethod
+    def from_issuer_public_key(
+        cls, public_key: Union[RSAPublicKey, DSAPublicKey, EllipticCurvePublicKey, Ed25519PublicKey, Ed448PublicKey]
+    ) -> AuthorityKeyIdentifier: ...
+    @classmethod
+    def from_issuer_subject_key_identifier(cls, ski: SubjectKeyIdentifier) -> AuthorityKeyIdentifier: ...
+
+class SubjectKeyIdentifier(ExtensionType):
+    digest: bytes
+    def __init__(self, digest: bytes) -> None: ...
+    @classmethod
+    def from_public_key(
+        cls, public_key: Union[RSAPublicKey, DSAPublicKey, EllipticCurvePublicKey, Ed25519PublicKey, Ed448PublicKey]
+    ) -> SubjectKeyIdentifier: ...
+
+class AccessDescription:
+    access_method: ObjectIdentifier
+    access_location: GeneralName
+    def __init__(self, access_method: ObjectIdentifier, access_location: GeneralName): ...
+
+class AuthorityInformationAccess(ExtensionType):
+    def __init__(self, descriptions: List[AccessDescription]) -> None: ...
```

Comment:
The List can be Iterable since the method calls `list()` on it on the first line. This also goes for the next class.

Reply:
Right, I wasn't sure because these are inconsistently typed in this file (List, Sequence and Iterable are all used). I originally went with `list` because the documentation says list. But I will change to Iterable. :+1: